http-client/src/main/java/no/nav/familie/http/config/RestTemplateBuilderBean.kt

```diff
@@ -1,18 +1,27 @@
 package no.nav.familie.http.config
 
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.boot.web.client.RestTemplateBuilder
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Import
+import org.springframework.context.annotation.Primary
 
 @Suppress("SpringFacetCodeInspection")
 @Configuration
 @Import(NaisProxyCustomizer::class)
 class RestTemplateBuilderBean {
 
     @Bean
+    @Primary
```

Comment:
Hvorfor trenger denne primary?

Reply:
Hvis du har konfigurert med proxy, så må den settes som primary, hvis ikke ender du op med to bønner av samme type.

Reply:
Ah ja, pga den andre beanen i denna fila. 
Men pga att denne er primary så får vi då ikke overstyrt den selv i applikasjonen?